src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -661,7 +652,7 @@ else if (KoLCharacter.inAxecore()) {
       return limit;
     }
 
-    if (KoLCharacter.isFullnessIncreased
+    if (HolidayDatabase.isFullnessIncreased()
         && (KoLCharacter.getPath() == Path.NONE || KoLCharacter.getPath() == Path.TEETOTALER)) {
```

Comment:
could you shift this check into `isFullnessIncreased` or change the method name to `isFeastOfBoris`? Just to make it a bit clearer?

Reply:
Ah right since I made a version that doesn't merge shared holidays this can just be renamed. Good idea!